core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeRelationService.java

```diff
@@ -31,14 +29,7 @@ public List<EnrolleeRelation> findByParticipantUserId(UUID participantUserId) {
         return dao.findByEnrolleeIdAndRelationshipType(enrolleeId, relationshipType);
     }
 
-    public List<Enrollee> findGovernedEnrollees(UUID participantUserId) {
-        List<EnrolleeRelation> enrolleeRelations=
-                findByParticipantUserId(participantUserId).stream().filter(enrolleeRelation -> isProxy(enrolleeRelation))
-                        .collect(Collectors.toList());
-        return enrolleeService.findAll(enrolleeRelations.stream().map(EnrolleeRelation::getId).toList());
-    }
-
```

Comment:
to avoid circular services, I broke this and moved part of it to EnrolleeService, it also makes more sense since it is supposed to return a list of Enrollees